### PR TITLE
ts_query_ls: init at 1.0.1

### PR DESCRIPTION
**Pull request by ribru17**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -18251,6 +18251,12 @@
     github = "ribose-jeffreylau";
     githubId = 2649467;
   };
+  ribru17 = {
```
Comment (FliegendeWurst) [inline maintainers/maintainer-list.nix:18254]: This should be a separate commit

**Review by FliegendeWurst — CHANGES_REQUESTED** [review]
Package looks good. Please make sure your commit messages follows CONTRIBUTING.md

**Comment by ribru17** [comment]
Thank you for the review :)

**Review by FliegendeWurst — APPROVED** [review]

**Comment by uncenter** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 350834`

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>ts_query_ls</li>
  </ul>
</details>

**Review by uncenter — APPROVED** [review]

**Review by isabelroses — APPROVED** [review]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 350834`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>ts_query_ls</li>
  </ul>
</details>

File: pkgs/by-name/ts/ts_query_ls/Cargo.lock
Comment (isabelroses) [inline pkgs/by-name/ts/ts_query_ls/Cargo.lock:1]: you don't need to vendor this anymore if you rebase such that you have this PR https://github.com/NixOS/nixpkgs/pull/349360 such that you can use `useFetchCargoVendor = true;`

**Review by isabelroses — COMMENTED** [review]
Also you can drop the `v` from the commit e.g. `ts_query_ls: init at 1.0.1`

File: pkgs/by-name/ts/ts_query_ls/Cargo.lock
Comment (ribru17) [reply pkgs/by-name/ts/ts_query_ls/Cargo.lock:1]: Awesome, thanks!

**Comment by uncenter** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 350834`

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>ts_query_ls</li>
  </ul>
</details>

**Review by uncenter — APPROVED** [review]

**Review by isabelroses — APPROVED** [review]
Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 350834`

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>ts_query_ls</li>
  </ul>
</details>

**Comment by uncenter** [comment]
For searchability do you want to update the PR title to match the commit message? `feat: ts_query_ls at v1.0.1` -> `ts_query_ls: init at 1.0.1`. Other than that this still looks good, hopefully we can get this merged!

**Review by wolfgangwalther — APPROVED** [review]

**Pull request merged**